tests/unit/sessions/test_models.py

```diff
@@ -78,33 +78,3 @@ def test_session_model(
 
     # Check that max_age is correctly parsed into a timedelta object
     assert session_direct.max_age == session_camel.max_age == session_snake.max_age == timedelta(minutes=30)
-
```

Comment:
With the new models, this didn't really test anything (except that assignment works..)